encryption/README.md

```diff
@@ -0,0 +1,54 @@
+# Encryption
+
+## What is it?
+
+Encryption involves complex mathematics to obscure data so that it is almost impossible to read without the correct decryption keys or passwords. End-to-end encryption is commonly used when transferring information that needs to be kept secret over a public system such as the internet.
+
+Critically, Encryption provides the following benefits:
+
+**Non-repudiation** - You can be sure that only the intended sender sent that message. Nobody else.  
+**Integrity** - The message has not been changed on the way to you by someone malicious.  
+**Confidentiality** - Nobody except you and the sender can understand the message.
+
+
+## Why is it important?  
+Without encryption the internet loses a lot of its functions...
+
+1. Home Finance  
+Without being able to confirm both parties' identities and communicate information securely, banks would unable to provide internet banking in the UK.
+
+2. Online Retail  
+Without being able to transfer payment details online securely, online retailers would no longer are able to sell to UK citizens without risking high levels of fraud. Your bank will also want to avoid insecure online payments as this exposes them to fraud.
+
+3. Secure Commmunication  
+Online messaging applications such as iMessage, WhatsApp and Facebook Messenger will not want to compromise their security for the sake of one nation and the UK could lose access to these platforms.
+
+4. All websites with logins  
+Logging into a website without encryption will expose your credentials to anyone who is connected to your network, either by wire or wirelessly. 
+
+5. Back to Cash  
+All card payments and ATMs use the internet to verify payments and balance. Like Online Retail, this is unusable and we go back to cash.
+
+### But what if we keep encryption, but let GCHQ have a backdoor?
+Allowing ANY single person to break encryption results in a breakdown of trust.
+
+Even if you take the position that a government will not abuse their position, the details on how to break it will eventually reach a hacker (as backdoors have done before). This would leave the entire UK population exposed to the largest vulnerability in the world.
+
+Even if you do trust our government with all your secure data, unless every other country trusts our government we're back to no access to the internet again.
+
+### What if we license encryption?
+Only allowing certain individuals to communicate securely is akin to licensing speech and would be a violation of human rights? //Todo: Be certain on this point. A right to privacy and freedom of expression **should** cover this?
```

Comment:
you could weasel-word it?
`and could constitute a violation of the right to privacy and freedom of expression`